### DISCUSSION TITLE
IP Block

### DISCUSSION THREAD

**Discussion by Coleslav** [macxq/foxess-ha#54] [Q&A]
Hello, i am wondering about adding this integration into my ha and I've got little question for you guys. As its written in readme "USING THIS INTEGRATION CAN CAUSE BLOCKING OF YOUR IP ADDRESS BY FOXESSCLOUD" I am wondering if its permanent block or its some kind of `flood protection` and after X minutes/hours my IP is going to be unblocked? 
Thanks !

**Comment by jsavoor** [comment]
Hey Devs .. I am also blocked from installing this. Can you please suggest a plan of action? Should I install or wait for you.
Thanks.
